Makefile

```diff
@@ -1,6 +1,7 @@
 # User-customizable variables:
 CXX ?= c++
-CXXFLAGS ?= -I relacy/fakestd -O1
+CXX_STD ?= c++11
```

Comment:
I might work on a CMake build to replace this, depending on how much more I work on relacy in the near term.